staging/src/k8s.io/legacy-cloud-providers/azure/azure_backoff.go

```diff
@@ -32,6 +33,10 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	vmssVMNotActiveErrorMessage = "not an active Virtual Machine Scale Set VM instanceId"
```

Comment:
what do you mean by "not active"? not exists?

Reply:
it looks like error msg return by azure API, may add comments about "not active" meaning.

Reply:
and this is only for VMSS? what about VMAS?

Reply:
Yep, the error message is coming from VMSS API, so it is VMSS only.

Reply:
Would add a comment for this.